### ISSUE TITLE
nuget command displays no output for requests to load a package that is already loaded

### ISSUE THREAD

**Issue by KevinRansom** [dotnet/interactive#1518]
This PR reduces the nuget displayed message: https://github.com/dotnet/interactive/pull/1510

However, it does not display output for requests that have already been loaded, producing a display like this:
![image](https://user-images.githubusercontent.com/5175830/127709632-34dc1c1a-1e7b-49ba-a450-49aa664ae4b4.png)

Note that the second cell requests:
````
#r "nuget: FSharp.Data"
#r "nuget: FSharp.Compiler.Service"
````

The output shows only:
![image](https://user-images.githubusercontent.com/5175830/127709879-9a2bccbe-5978-4ac7-89bf-c270c9c2a040.png)

This can be confusing to the user, did it or did it not load FSharp.Compiler.Service, why can't I see it? What did I do wrong?

In the image above it is clear what has happened but what about this one:
![image](https://user-images.githubusercontent.com/5175830/127710309-dbe01b62-215e-479f-b6db-9d624837ed98.png)

I doubt anyone would be explain why google.protobuf isn't showed as loaded, without inferring that it was a dependency of OnnxTransformer and so had already been loaded.

I propose that we change the display to always show the set of requested packages for the submission that caused the Restore.

**Comment by KevinRansom** [comment]
Okay so I have changes that produce this:

![image](https://user-images.githubusercontent.com/5175830/128072350-bc70fe29-75a0-42be-b33c-1c467fdf9b83.png)


![image](https://user-images.githubusercontent.com/5175830/128072650-ddf46d31-4788-4e6a-8f58-ab42f05539a5.png)

**Comment by colombod** [comment]
I think this is a good user experience improvement, is there a PR open?

**Comment by brettfo** [comment]
Fixed in #1529.
